finalPrep/modZeroHero.js

```diff
@@ -2,40 +2,86 @@
 
 // Declare two variables - heroName AND specialAbility - set to strings
 
+var heroName = "Superman";
+var specialAbility = "super strength";
+
 // Declare two variables - greeting AND catchphrase
 //   greeting should be assigned to a string that uses concatenation to include the heroName
 //   catchphrase should be assigned to a string that uses interpolation to include the specialAbility
 
+var greeting = "It's a bird.  It's a plane.  It's " + heroName + "!";
+var catchphrase = `Superman is known for his ${specialAbility}.`;
+
 // Declare two variables - power AND energy - set to integers
 
+var power = 9001;
+var energy = 100;
+
 // Declare two variables - fullPower AND fullEnergy
 //   fullPower should multiply your current power by 500
 //   fullEnergy should add 150 to your current energy
 
+var fullPower = power * 500;
+var fullEnergy = energy + 150;
+
 // Declare two variables - isHuman and identityConcealed - assigned to booleans
 
+var isHuman = false;
+var identityConcealed = true;
 
-// Declare two variables - archEnemies AND sidekicks
+
+// Declare two variables - archEnemies AND sideKicks
 //   archEnemies should be an array of at least 3 different enemy strings
 //   sidekicks should be an array of at least 3 different sidekick strings
 
+var archEnemies = ["Lex Luther", "Brainiac", "Zod"];
+var sideKicks = ["Super Dog", "Super Girl", "Lois Lane"];
+
 // Print the first sidekick to your console
 
+console.log(sideKicks[0]);
+
 // Print the last archEnemy to the console
 
+console.log(archEnemies[2]);
```

Comment:
How would you print the last `archEnemy` to the console if you didn't know the length of the array?